back/app/services/permissions/user_requirements_service.rb

```diff
@@ -237,4 +121,4 @@ def app_configuration
   end
 end
 
-PermissionsService.prepend(Verification::Patches::PermissionsService)
+Permissions::UserRequirementsService.prepend(Verification::Patches::Permissions::RegistrationRequirementsService)
```

Comment:
Should it be named the same as the main file?
```suggestion
Permissions::UserRequirementsService.prepend(Verification::Patches::Permissions::UserRequirementsService)
```

Reply:
Yes!